pkg/controller/projectclaim/projectclaimadapter.go

```diff
@@ -41,6 +42,7 @@ const (
 const ProjectClaimFinalizer string = "finalizer.gcp.managed.openshift.io"
 const CCSSecretFinalizer string = "finalizer.gcp.managed.openshift.io/ccs"
 const RegionCheckFailed string = "RegionCheckFailed"
+const FakeProjectClaim string = "managed.openshift.com/fake"
```

Comment:
These fake Projects are going to be used for hive performance testing. 
Who is setting this annotation for fake ProjectClaims that are generated for fake ClusterDeployments? Is it a hive?
How did we come up with this naming? 
Shouldn't it be the same as here? : 
https://github.com/openshift/hive/blob/023f095eaf99afeb730b103f6881ef081c7ac88e/pkg/constants/constants.go#L321

Reply:
Confirmed with Manuel that `managed.openshift.com/fake` is correct.